### PR TITLE
Fix peaks error

### LINKED ISSUE #173: Upgrade Peaks.js in SME
## Description

SME uses `v0.26.0` of Peaks.js library. The latest stable version is `2.1.0` and they are working on a `3.0.0` which is in a beta release now. 

## Done Looks Like
- [ ] Upgrade `Peaks.js` and any other dependencies
- [ ] All SME functions work without breaking

### PR DESCRIPTION
**Pull request by Dananji**
Fix peaks.js initialization error with the upgrade from 0.26.0 -> 2.1.0

Closes #173

### CODE REVIEW

File: src/components/Waveform.js [hunk 1]
```diff
@@ -85,18 +83,9 @@ const Waveform = React.forwardRef((props, ref) => {
...
-      dispatch(
-        retrieveStreamMedia(audioFile, ref.mediaPlayerRef.current, {
-          withCredentials: props.withCredentials,
-        })
-      );
-    }
-  }, [readyPeaks]);
+  // React.useEffect(() => {
+  //   setPeaksIsReady(readyPeaks);
+  // }, [readyPeaks]);
```
Comment (cjcolvar) [inline src/components/Waveform.js:88]: Can this be removed?

**Review by cjcolvar — APPROVED** [review]
Looks good.  I only had a couple of non-blocking questions.

File: src/actions/peaks-instance.js [hunk 2]
```diff
@@ -4,108 +4,62 @@ import Peaks from 'peaks.js';
...
     let waveformInfo;
 
-    try {
-      const response = await apiUtils.getRequest(manifestURL);
+    const { manifest } = getState();
 
-      if (!isEmpty(response.data)) {
-        mediaInfo = getMediaInfo(response.data, canvasIndex);
-        waveformInfo = getWaveformInfo(response.data, canvasIndex);
+    if(manifest) {
```
[the remark this answers is not in the document]
Comment (Dananji) [reply src/actions/peaks-instance.js:36]: I think the space between is preferred when it comes to formatting. I had a set of formatting rules specified in my VSCode, but lost them when I re-installed it after formatting my machine. Thanks for noticing this, I will get on fixing the formatting :100:

[the remark this answers is not in the document]
Comment (Dananji) [reply src/actions/peaks-instance.js:36]: I should probably re-initialize my formatting settings in VSCode and take care of these inconsistencies with spacing and other formatting related things.

**Pull request merged**